Survey Sparrow - New Components

michelle0927 opened the pull request:
Resolves #6784 

Trigger
New Survey Response

Actions
Create Contact
Create Survey
Update Survey

lcaresia reviewed (approved):
LGTM!

vunguyenhung commented:
Hello everyone, I have tested this PR and there're some test cases failed or needed improvement. 

Please check the test report below for more information
[SurveySparrow_6784_0010.pdf](https://github.com/PipedreamHQ/pipedream/files/11772624/SurveySparrow_6784_0010.pdf)

michelle0927 commented:
@vunguyenhung I had assumed an "email theme" was what they wanted for theme_id, but it seems like that's incorrect and I'm not sure how to find theme_ids, so I removed it. I removed `visibility` from `update-survey`, and removed `theme_id` from `update-survey` and `create-survey`.

vunguyenhung commented:
Hi everyone, all test cases are passed! Ready for release! 

Test report
[SurveySparrow_6784_1428.pdf](https://github.com/PipedreamHQ/pipedream/files/11780840/SurveySparrow_6784_1428.pdf)

michelle0927 commented:
/approve
